src/client/utils/validations.js

```diff
@@ -468,7 +475,6 @@ export {
   isValidMonetaryValue,
   isValidPhone,
   isValidEmail,
-  isValidAddress,
```

Comment:
`isValidAddressField` is called via `isValidForm`, which is called when the 'Continue' button is clicked